developer/src/kmc/src/commands/build.ts

```diff
@@ -1,8 +1,8 @@
 import * as fs from 'fs';
 import { Command } from 'commander';
-import { BuildActivityOptions } from '../activities/BuildActivity.js';
-import { buildActivities } from '../activities/buildActivities.js';
-import { BuildProject } from '../activities/BuildProject.js';
+import { BuildActivityOptions } from './build/BuildActivity.js';
+import { buildActivities } from './build/buildActivities.js';
```

Comment:
Maybe this was asked before, is there a reason `buildActivities.ts` isn't CamelCase like all the other files?(buildTestData.ts too)

Reply:
One is a module containing a class, the other is a module containing a variable.